src/filter/tree.rs

```diff
@@ -254,6 +254,73 @@ pub fn overlay(
     return Ok(input1);
 }
 
+pub fn pathline(b: &str) -> JoshResult<String> {
+    for line in b.split("\n") {
```

Comment:
don't you just want to do it for the first line? Why iterate?

Reply:
Because it is easier to write this way ;)